src/jobs/looksrare-v2-sync/utils.ts

```diff
@@ -188,3 +189,187 @@ export const fetchOrders = async (
 
   return [mostRecentCreatedHash, ""];
 };
+
+export const fetchSeaportOrders = async (
+  lastSyncedHash: string = "",
+  cursor: string = "",
+  startTime: number = 0,
+  endTime: number = 0,
+  backfill = false
+) => {
+  logger.info(
+    "fetch_seaport_orders_looksrare_v2",
+    `lastSyncedHash = ${lastSyncedHash}, cursor = ${cursor} Fetching Seaport orders from LooksRareV2`
+  );
+
+  const looksRare = new LooksRareV2();
+  const seaport = new Seaport();
+  let limit = 150;
+  let maxOrdersToFetch = 1000;
+  let mostRecentCreatedHash: string = "";
+
+  let numOrders = 0;
+
+  let done = false;
+  while (!done) {
+    const url = looksRare.buildFetchOrdersURL(
+      {
+        startTime,
+        endTime,
+      },
+      {
+        limit,
+        cursor,
+      },
+      true
+    );
+
+    try {
+      const response = await axios.get(
+        url,
+        config.chainId === 1
+          ? {
+              headers: {
+                "X-Looks-Api-Key": config.looksrareApiKey,
+                "user-agent":
+                  "Mozilla/5.0 (X11; Fedora; Linux x86_64; rv:89.0) Gecko/20100101 Firefox/89.0",
+              },
+              timeout: 10000,
+            }
+          : // Skip including the API key on Rinkeby or else the request will fail
+            {
+              headers: {
+                "X-Looks-Api-Key": config.looksrareApiKey,
+              },
+              timeout: 10000,
+            }
+      );
+
+      const orders: LooksRareSeaportOrder[] = response.data.data;
+      const parsedOrders: {
+        kind: "seaport-v1.4" | "seaport-v1.5";
+        data: Sdk.SeaportBase.Types.OrderComponents;
+      }[] = [];
+
+      const values: any[] = [];
+      const handleOrder = async (order: LooksRareSeaportOrder) => {
+        order.protocol_address = Sdk.SeaportV15.Addresses.Exchange[config.chainId];
+        const parsed = await seaport.parseSeaportOrder(order);
+        if (parsed) {
+          parsedOrders.push({
+            kind: parsed.kind,
+            data: parsed.order.params as any,
+          });
+        }
+
+        values.push({
+          hash: order.hash.toLowerCase(),
+          target:
+            parsed?.order.getInfo()?.contract.toLowerCase() ||
+            order.protocol_data.parameters.offer[0].token.toLowerCase(),
+          maker: order.protocol_data.parameters.offerer.toLowerCase(),
```

Comment:
@georgeroman Not sure if this is valid, I assume offerer is equivalent to maker but didn't see `maker` anywhere explicitly

Reply:
That's right, offerer = maker